### PR TITLE
Set the cut to reject events with nearby events for ITS-TPC m.e. study

### PR DESCRIPTION
**Pull request by ChunzhengLab**

### CODE REVIEW

**Review by iouribelikov — APPROVED** [review]

File: DPG/Tasks/AOTTrack/qaMatchEff.cxx [hunk 1]
```diff
@@ -3363,6 +3364,13 @@ struct qaMatchEff {
         return;
       }
     }
+    if (isRejectNearByEvent) {
+      if (!collision.selection_bit(o2::aod::evsel::kNoCollInTimeRangeStandard)) {
```
Comment (mfaggin) [inline DPG/Tasks/AOTTrack/qaMatchEff.cxx:3368]: @ChunzhengLab are you sure that you have to put `!` in front? From slide 22 here https://indico.cern.ch/event/1425523/contributions/5995739/attachments/2893781/5081523/performance_vs_occupancy_AW_July_10_2024_IA_updated.pdf it seems to me that this is not the case. Can you confirm it with Igor?

File: DPG/Tasks/AOTTrack/qaMatchEff.cxx [hunk 2]
```diff
@@ -87,14 +87,15 @@ struct qaMatchEff {
...
   Configurable<bool> isEnableOccupancyCut{"isEnableOccupancyCut", false, "Boolean to switch the occupancy cut on/off."};
   struct : ConfigurableGroup {
     Configurable<float> centralityMinCut{"centralityMinCut", 0.0f, "Minimum centrality"};
     Configurable<float> centralityMaxCut{"centralityMaxCut", 100.0f, "Maximum centrality"};
   } centralityCuts;
   struct : ConfigurableGroup {
-    Configurable<int> minTracksInTimeRange{"minTracksInTimeRange", 0, "Minimum number of tracks in the time range"};
-    Configurable<int> maxTracksInTimeRange{"maxTracksInTimeRange", 1000, "Maximum number of tracks in the time range"};
+    Configurable<float> minTracksInTimeRange{"minTracksInTimeRange", 0.0f, "Minimum number of tracks in the time range"};
+    Configurable<float> maxTracksInTimeRange{"maxTracksInTimeRange", 999999.9f, "Maximum number of tracks in the time range"};
```
Comment (mfaggin) [inline DPG/Tasks/AOTTrack/qaMatchEff.cxx:98]: what's the reason to change the data type here?

File: DPG/Tasks/AOTTrack/qaMatchEff.cxx (hunk 1 above)
Comment (ChunzhengLab) [reply DPG/Tasks/AOTTrack/qaMatchEff.cxx:3368]: Hi Mattia, I have confirmed with Igor, this is correct. Put a `!` to reject events with the nearby event.

File: DPG/Tasks/AOTTrack/qaMatchEff.cxx (hunk 2 above)
Comment (ChunzhengLab) [reply DPG/Tasks/AOTTrack/qaMatchEff.cxx]: The default type of occupancy should indeed be int, and this has been corrected in the latest commit. 
The default maximum cut was too small and has been changed to a larger value.

**Review by mfaggin — APPROVED** [review]

**Pull request merged**